x-pack/plugins/apm/server/routes/create_api/index.ts

```diff
@@ -140,6 +140,7 @@ export function createApi() {
                   // Only return values for parameters that have runtime types,
                   // but always include query as _debug is always set even if
                   // it's not defined in the route.
+                  // @ts-ignore
                   params: pick(parsedParams, ...Object.keys(params), 'query'),
```

Comment:
@dgieselaar Any way to avoid `ts-ignore`?

This is the error I get without it
![image](https://user-images.githubusercontent.com/209966/85833971-e26b1c00-b792-11ea-92e9-3e28ed91afa0.png)

Reply:
not right away. I'm fine with merging as-is and us addressing it in a separate PR. It's not the _worst_ place to leave a ts-ignore.

Reply:
@sqren are we good to move on with the `ts-ignore`?

Reply:
Sure 👍